src/main/scala/ai/eto/rikai/sql/model/Model.scala

```diff
@@ -48,6 +49,10 @@ object Model {
       throw new ModelNameException(s"Model name '${name}' is not valid")
     }
   }
+
+  def serializeOptions(options: Map[String, String]): String = {
+    options.asJson.asString.getOrElse("")
```

Comment:
`asJson` was from `io.circe.syntax._`?  I dont have much scala experience . @changhiskhan what do you think?

Reply:
From the example of [circ documentation](https://circe.github.io/circe/codec.html), it is from `io.circe.syntax._`. @changhiskhan would like to learn from you.